layouts/partials/head.html

```diff
@@ -2,9 +2,23 @@
   <meta charset="utf-8">
   <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
   <meta name="viewport" content="width=device-width">
+
   <meta name="project" content="{{ .Params.project }}">
   <meta name="version" content="{{ .Params.project_version }}">
-  <meta name="base-url" content="/{{ .Scratch.Get "site_relative_url" }}">
+
+  <!-- TODO: Adding these leading slashes is... pretty dumb. IMO. We should
```

Comment:
Leading slashes in the sense of having to put /riak vs riak/ >>>project<<</version?